packages/core/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@amqp-contract/core",
+  "version": "0.1.0",
```

Comment:
follow the same version as other packages and add it to fixed version list un changset config